containers/kraken.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kraken-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      tool: Kraken
+  template:
+    metadata:
+      labels:
+        tool: Kraken
+    spec:
+      serviceAccountName: useroot
+      containers:
+        - name: kraken
+          securityContext:
+            privileged: true
+          image: quay.io/openshift-scale/kraken
+          command: ["/bin/sh", "-c"]
+          args: ["python3 run_kraken.py -c config/config.yaml"]
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - mountPath: "/root/.kube"
+              name: config
+            - mountPath: "/root/kraken/config"
+              name: kraken-config
+      volumes:
```

Comment:
The `scenarios` directory has to be mounted as the user places all the scenarios here.
After the node scenarios PR is merged, the `node scenarios` directory has to mounted as well. And the code has to be modified to make sure the node chosen for node scenario injection doesn't have kraken pod running on it.
Instructions regarding modifying pod and node scenarios in `scenarios` and `node scenarios` directories has to be added in the README.md

Reply:
I have added a mount for the scenarios directory. How do we want to proceed about the 2 coming stories that both add directories (recovered components and node failures scenarios)? Should we add the necessary mounts to each pull requests if this pull request gets merged first? 

Also, do I need to change anything on my side so the node kraken gets deployed on doesn't get killed during the scenario injections or will that be up to the user to properly specify nodes in their yaml files that don't include the one kraken is running on?

Reply:
You need to make change in this function: https://github.com/openshift-scale/kraken/pull/10/files#diff-54651a20a8f57ef1c067dbaac8fdc0a6R62 to make sure the selected node doesn't have kraken pod running on it.

Reply:
Maybe organize all the scenarios/post actions under scenarios directory, this way we can just mount one directory?

Reply:
> Maybe organize all the scenarios/post actions under scenarios directory, this way we can just mount one directory?

Updated other pull request

Reply:
> You need to make change in this function: https://github.com/openshift-scale/kraken/pull/10/files#diff-54651a20a8f57ef1c067dbaac8fdc0a6R62 to make sure the selected node doesn't have kraken pod running on it.

Added code for this in the kraken/kubernetes/client, please take a look